index.html

```diff
@@ -3,9 +3,84 @@
+            --><a href="projects.html">Projects</a>
+        </nav>
+    </header>
+    <section class="main">
+        <div class="img">
+            <img src="assets/andy-dutton-p9Hk-cbmIHo-unsplash.jpg" alt="Silhouette with blue sky">
+        </div>
+        <div class="blurb">
+            <h1>JOSH GLANTZ</h1>
+            <a href="projects.html">My Projects</a>
+            <p>Life brings us together from many disparate places and relishes in how the edges overlap, merge, and create new spaces. Coding is nowhere near where I thought I'd find myself one day, but it's been such a great adventure to find myself here. I've been a pre-med student, personal chef, capoeira instructor, olive oil store manager, home renovator & small group leader. Different circles know me as the laid-back trickster, quiet bibliophile, curious food finder, enthusiastic amatuer permaculturalist & aspiring world traveler. Here's to discovering how your path and mine can cross & create something new together.</p>
+        </div>
+    </section>
+    <section class="gallery">
+        <div>
+            <a href="https://josh-glantz-created.netlify.app/first-website/">
+                <img src="assets/First Website Screenshot.png" alt="First Website screenshot">
+                <p>OC: Chicken Adobo</p>
+            </a>
+        </div>
+        <div>
+            <a href="https://josh-glantz-created.netlify.app/shayhoweclass/styles-conference/">
+                <img src="assets/Screenshot of Styles Conference.png" alt="Screenshot of Styles Conference">
+                <p>Styles Conference Remake</p>
+            </a>
+        </div>
+        <div>
+            <a href="https://joshs-guide-to-yosemite.netlify.app/">
+                <img src="assets/Guide to Yosemite Screenshot.png" alt="Guide to Yosemite screenshot">
+                <p>Guide to Yosemite Remake</p>
+            </a>
+        </div>
+        <div>
+            <!-- <figure> -->
+                <img src="assets/tim-cooper-XfqGuWevmvs-unsplash.jpg" alt="Lush garden archway">
+            <!-- </figure> -->
+            <p>Todo: Custom Plant Guilds</p>
+        </div>
+        <div>
+            <img src="assets/med-badr-chemmaoui-ZSPBhokqDMc-unsplash.jpg" alt="Filled journal">
+            <p>Todo: Design Your Life</p>
+        </div>
+        <div>
+            <img src="assets/dhruv-patel-xiSBJmn79hk-unsplash.jpg" alt="Organized closet">
+            <p>Todo: Closet Space</p>
+        </div>
+        <div>
+            <img src="assets/kevin-luarte-FHTxbpRWEsE-unsplash.jpg" alt="Badges on messenger bag">
+            <p>Todo: Permies.com PEP Badge Tracker</p>
+        </div>
+        <div>
+            <img src="assets/amandine-p-oX3ElhukPIU-unsplash.jpg" alt="Coins found along Kumano Kodo">
+            <p>Todo: Kumano Kodo Concierge</p>
+        </div>
+    </section>
+    <footer>
+        <h4>Like what you see?</h4>
+        <a href="mailto:josh.glantz@gmail.com" class="mail">Contact Me</a>
```

Comment:
I would set up a new email specifically for this. Putting your email out publicly like this will attract ungodly amounts of spam.

Reply:
🤦